lib/localparticipant.js

```diff
@@ -585,6 +591,7 @@ class LocalParticipant extends Participant {
  * A {@link LocalTrack} was removed by the {@link LocalParticipant}.
  * @param {LocalTrack} track - The {@link LocalTrack} that was removed
  * @event LocalParticipant#trackRemoved
+ * @deprecated
```

Comment:
What do we suggest instead? Do we recommend not using this event at all?

Reply:
@markandrus "trackAdded" and "trackRemoved" in `LocalParticipant` do not serve any purpose, at least in v2.0.